Code/Tools/ProjectManager/Platform/Linux/ProjectManagerDefs_linux.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectManagerDefs.h>
+
+
+namespace O3DE::ProjectManager
+{
+    static const QString ProjectBuildPathPostfix = ProjectBuildDirectoryName + "/linux";
```

Comment:
How about `linux_make` for the build path to distinguish it from the ninja generator in case the user is using that one.

Reply:
we use linux for Ninja already in Jenkins
Since we support just one generator per platform (except for windows where we have windows_vs2019), linux makes it consistent with ci_build scripts